compiler/modules/print.llvm/llvm.bal

```diff
@@ -343,38 +406,38 @@ public class Builder {
     }
 
     // Corresponds to LLVMBuildAlloca
-    public function alloca(IntegralType ty, Alignment? align = (), string? name=()) returns PointerValue {
+    public function alloca(IntegralType ty, Alignment? align=(), string? name=()) returns PointerValue {
         BasicBlock bb = self.bb();
         string reg = bb.func.genReg(name);
         PointerType ptrTy = pointerType(ty);
-        addInsnWithAlign(bb, [reg, "=", "alloca", typeToString(ty)], align);
+        addInsnWithAlign(bb, [reg, createLine(["=", "alloca", typeToString(ty)])], align);
         return new PointerValue(ptrTy, reg);
     }
 
     // Corresponds to LLVMBuildLoad
-    public function load(PointerValue ptr, Alignment? align = (), string? name=()) returns Value {
+    public function load(PointerValue ptr, Alignment? align=(), string? name=()) returns Value {
         BasicBlock bb = self.bb();
         Type ty = ptr.ty.pointsTo;
         string reg = bb.func.genReg();
-        addInsnWithAlign(bb, [reg, "=", "load", typeToString(ty), ",", typeToString(ptr.ty), ptr.operand], align);
+        addInsnWithAlign(bb, [reg, createLine(["=", "load", typeToString(ty), ",", typeToString(ptr.ty)]), ptr.operand], align);
         return new Value(ty, reg);
     }
 
     // Corresponds to LLVMBuildStore
-    public function store(Value val, PointerValue ptr, Alignment? align = ()) {
+    public function store(Value val, PointerValue ptr, Alignment? align=()) {
         Type ty = ptr.ty.pointsTo;
         if ty != val.ty {
             panic error("store type mismatch: " + typeToString(val.ty) + ", " + typeToString(ptr.ty));
         }
-        addInsnWithAlign(self.bb(), ["store", typeToString(ty), val.operand, ",", typeToString(ptr.ty), ptr.operand], align);
+        addInsnWithAlign(self.bb(), [createLine(["store", typeToString(ty)]), val.operand, createLine([",", typeToString(ptr.ty)]), ptr.operand], align);
```

Comment:
Adding all these calls to `createLine` is not a good solution to the problem.

Reply:
Replaced with new function `concat` that properly handle merging (string|int) chunks and removed changes to createLine